src/utils.coffee

```diff
@@ -214,6 +214,19 @@ exports.getKnexApp = (appId, columns) ->
 			throw new AppNotFoundError('App not found')
 		return app
 
+exports.getDeviceConfig = ->
```

Comment:
I think it'd make sense for these to be under `device.coffee`, ie `device.getConfig()`/`device.setConfig()`